src/include/rbd/io_operations.h

```diff
@@ -0,0 +1,26 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
```

Comment:
Nit: since the contents of this directory will be shipped with the `librbd-devel` package, I just wonder if this really needs to be created and instead (1) re-use the enums already created in `librbd/io/Types.h` and (2) move the remainder of logic to `librbd/io/IoOperations.cc`

Reply:
Sorry for the legacy code, updated with (2) move the remainder of logic to librbd/io/IoOperations.cc

Reply:
Hmm, this file is still here?

Reply:
Updated, move content of src/include/rbd/io_operations.h to librbd/io/Types.h and librbd/io/IoOperations.cc